<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>minHS</class>
 <widget class="QWidget" name="minHS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>minHS</string>
  </property>
  <widget class="QPushButton" name="push_button_start">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_stop">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>PLAY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>279</width>
     <height>69</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
background-color: rgb(181, 181, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>Not Running.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_restore_HS">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Restore
Hearthstone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_restore_XD">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Restore
XD.exe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_restore_battleNet">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Restore
Battle.net</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_killall3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>KILL ALL
(3)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_MINIMIZE_Xshell">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>MINIMIZE
Xshell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_MINIMIZE_Self">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>81</width>
     <height>69</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>MINIMIZE
SELF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_start_helperexe">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Start
Helper.exe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_MINIMIZE_Chrome">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>MINIMIZE
Chrome</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
